docs/deploy-from-source-code.md

```diff
@@ -0,0 +1,94 @@
+# Deploy from source code
+
+## Platform from source code getting started
+
+- Get the latest platform source code from [release/3.0.0](https://github.com/VirtoCommerce/vc-platform/tree/release/3.0.0)
+- Set public url for assets `Assets:FileSystem:PublicUrl` with url of your application, this step is needed in order for display images
+
+```json
+"Assets": {
+        "Provider": "FileSystem",
+        "FileSystem": {
+            "RootPath": "~/assets",
+            "PublicUrl": "http://localhost:10645/assets/" <-- Set your platform application url with port localhost:10645
+        },
+     
+    },
+```
+
+- Open `VirtoCommerce.Platform.sln` solution in Visual Studion 2019 and press F5 or run via `dotnet` CLI by typing in the console the follow commands
+
+```console
+cd src\VirtoCommerce.Platform.Web
+```
+
+- Install all required npm packages
+
+   ```console
+    npm ci
+   ```
+
+- Bundle all js scripts and css styles
+
+```console
+npm run webpack:build
+```
+
+- Run platform by dotnet CLI.
+
+**Note:** you can add `--no-build` flag to speed up start if you already compile solution.
+
+```console
+dotnet run -c Development --no-launch-profile
+```
+
+- Open in your browser follow url `http://localhost:10645`.
```

Comment:
Open `https://localhost:5001` in the browser.

Reply:
fixed, but in Development mode used http and 10645 port